### PR TITLE
Added RebaiAhmed.json 

### PR DESCRIPTION
**Pull request by Rebaiahmed**
Adding myself to the Contributor list

### CODE REVIEW

File: src/contributors/RebaiAhmed.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Rebai Ahmed",
+    "desc": "I am Full Stack Developer ,Open source Enthusiast , TunisianJs-Community Co-Founder , Meetups Co-Organizer",
+    "github": "https://github.com/Rebaiahmed",
+    "twitter": "https://twitter.com/RebaiAhmed_",
+    "linkedin": "https://www.linkedin.com/in/rebaiahmed",
+    "stackoverflow": "https://stackoverflow.com/users/5330655/rebai-ahmed",
+    "quora": "https://www.quora.com/profile/Rebai-Ahmed",
+    "medium" :"https://ahmedrebai.medium.com",
+    "portfolio":"https://rebaiahmed.github.io"
```
Comment (iamdarshshah) [inline src/contributors/RebaiAhmed.json:10]:
```suggestion
    "linkedin": "https://www.linkedin.com/in/rebaiahmed"
```
Thanks for adding all of your socials, appreciated. Unfortunately, we have options to just add `github`, `twitter` & `linkedin`. Please update it

**Review by iamdarshshah — CHANGES_REQUESTED** [review]

File: src/contributors/RebaiAhmed.json (hunk 1 above)
Comment (Rebaiahmed) [reply src/contributors/RebaiAhmed.json]: I just updated it :)

**Review by iamdarshshah — APPROVED** [review]
This now LGTM! Thanks @Rebaiahmed

**Comment by Rebaiahmed** [comment]
> This now LGTM! Thanks @Rebaiahmed

Please , can you add Hacktoberfet label to my PR ?

**Comment by iamdarshshah** [comment]
Hey, @Rebaiahmed there is no need for that as of now. As PR is already merged and it will be counted as valid. 😄

**Pull request merged**